pytest_fixtures/api_fixtures.py

```diff
@@ -323,9 +325,14 @@ def module_libvirt_image(module_cr_libvirt):
 
 
 @pytest.fixture(scope='session')
-def googleclient():
+def gce_cert():
```

Comment:
Bit of overkill actually making this a fixture, I'd probably just do the `json.loads` in the `googleclient` fixture itself.

No harm really, but its best for fixtures to actually take some 'action' to effect test setup. `googleclient` fixture is creating the instance - but its also calling `download_gce_cert`.